src/providers/SPWebContext.ts

```diff
@@ -1,27 +1,17 @@
-import { spfi, SPFI, SPBrowser } from "@pnp/sp";
-//import { LogLevel, PnPLogging } from "@pnp/logging";
+import { spfi, SPBrowser } from "@pnp/sp";
 import "@pnp/sp/webs";
-import "@pnp/sp/lists";
-import "@pnp/sp/items";
-import "@pnp/sp/batching";
-import "@pnp/sp/site-users/web";
-import "@pnp/sp/profiles";
```

Comment:
These are used by the PeoplePicker and UserApi so need to remain otherwise the global context will not function correctly.   Ideally we maybe create separate connections but for now it is using global.